### PR TITLE
Better style and popup management for editors

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15855/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15855/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15855/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15855/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15855/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15855/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15855/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15855/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15855/merge/testResults/webgpuplaywright/index.html

File: packages/dev/sharedUiComponents/src/popupHelper.tsx
[the remark this answers is not in the document]
Comment (deltakosh) [reply packages/dev/sharedUiComponents/src/popupHelper.tsx:1]: Good catch!

**Comment by deltakosh** [comment]
@AmoebaChant for the popup removal for Smart Filters

**Pull request merged**